packages/gatsby/src/utils/api-browser-docs.js

```diff
@@ -59,6 +59,7 @@ exports.onRouteUpdate = true
  * @param {object} $0
  * @param {object} $0.prevRouterProps The previous state of the router before the route change.
```

Comment:
I feel that the description of `prevRouterProps` is a little misleading, especially when combined with my introduction of `state`.

Is it really "the previous state of the router", given that we have an actual property named `state` in the `prevRouterProps.location` object?